test/functional/services/remote/webdriver.ts

```diff
@@ -111,7 +114,10 @@ function initChromiumOptions(browserType: Browsers, acceptInsecureCerts: boolean
 
   if (remoteDebug === '1') {
     // Visit chrome://inspect in chrome to remotely view/debug
-    options.headless();
+
+    // Using the new headless mode (instead of `options.headless()`)
+    // See: https://www.selenium.dev/blog/2023/headless-is-going-away/
+    options.addArguments('headless=new');
```

Comment:
If the file is not downloaded, we might need to try passing download settings via experimental options:

```
{
  "goog:chromeOptions": {
    "experimentalOptions": {
          ...
      }
}